patterns/sidebar-writer.php

```diff
@@ -17,7 +17,7 @@
 <!-- /wp:heading -->
 
 <!-- wp:paragraph {"style":{"typography":{"fontSize":"0.9rem"}}} -->
-<p style="font-size:0.9rem"><?php echo esc_html__( 'Je suis Adele Diouf, et je suis animée par une curiosité insatiable de dévoiler la tapisserie complexe de la finance et de l\'économie. Au cours de plus d\'une décennie, mon voyage à travers le labyrinthe des marchés mondiaux, des politiques monétaires et des tendances économiques a fourni aux lecteurs les outils nécessaires pour naviguer dans le paysage dynamique de la finance.', 'twentytwentyfour' ); ?></p>
+<p style="font-size:0.9rem"><?php echo esc_html_x( 'Je suis Adele Diouf, et je suis animée par une curiosité insatiable de dévoiler la tapisserie complexe de la finance et de l\'économie. Au cours de plus d\'une décennie, mon voyage à travers le labyrinthe des marchés mondiaux, des politiques monétaires et des tendances économiques a fourni aux lecteurs les outils nécessaires pour naviguer dans le paysage dynamique de la finance.', 'Description de l\'écrivain de la barre latérale', 'twentytwentyfour' ); ?></p>
```

Comment:
Is it necessary to write the context in French in this case? I'm wondering if there are guidelines for the language used for Translator Context.

Reply:
@vcanales I have Updated context Text for patterns/sidebar-writer.php 

Let me know if any correction / changes.

Reply:
The text should be in English.

Reply:
@carolinan whole paragraph should be in english ?  or just **"Description de l\'écrivain de la barre latérale"**

Reply:
Every single word that are not names.

Reply:
@carolinan 
I have added all paragraph text in English let me know if any correction